Create Activity\Visit class

Lehersun opened the pull request:
Implement the Activity\Visit class including the following state:

    day and time of the visit - /DateTime object
    status: planned, confirmed - string
    visitor: who will come - Core/Visitor object
    activity: what activity will be visited - Core/Activity object

Lehersun committed:
Create Activity\Visit class

Magomogo commented:
Tests first! Did you forget?

Magomogo commented on src/Activity/Visit.php:
```diff
@@ -0,0 +1,56 @@
+    public function getVisitor(): Visitor
+    {
+        return $this->visitor;
+    }
```
Please don't introduce any public interface until it's needed. I mean all those getters.

Lehersun replied:
I wrote this part in the hope I understand what it should return. I think I shouldn't have pushed it. Tests first, I remember

Lehersun committed:
Create Activity/Visit test

Assume that thw Visit class checks supposed visit time with the activity
schedule, checks the purchased subscription and returns the 'confirmed'
status if it matches

Lehersun committed:
Remove unnecessery funtionality from Visit object
